packages/app/src/settings/project/renderers/RenderObject.vue

```diff
@@ -46,10 +46,9 @@
     <span
       :class="props.colorClasses"
       :data-cy-config="props.from"
-    >{</span><br><template
-      v-for="(val, k) in props.value"
-      :key="k"
-    >
+    >{</span>
+    <br v-if="!Object.keys(props.value).length">
+    <template v-for="(val, k) in props.value" :key="k">
```

Comment:
`props.value` isn't always an `object` so this logic is breaking how non-empty sections are rendering in some situations. You can see the Typescript type for `value` in the prop definitions `Record<string, any> | any[]`

**Before**
![Screen Shot 2022-12-08 at 1 26 49 PM](https://user-images.githubusercontent.com/11482842/206550315-6b58948b-8c0c-41e3-a566-f04fd0b04ff5.png)

**After**
![Screen Shot 2022-12-08 at 1 26 40 PM](https://user-images.githubusercontent.com/11482842/206550344-3f68abd8-c02b-4865-8230-6c40fb699167.png)